### ISSUE TITLE
Set up highfive for this repo

### ISSUE THREAD

**Issue by fitzgen** [rust-lang-nursery/rust-wasm#45]
To automatically choose a reviewer from the peers when a new PR is made.

**Comment by fitzgen** [comment]
From `#rust-infra`:

```
pietroalbini> fitzgen, you should add a config file named rust-wasm.json in https://github.com/rust-lang-nursery/highfive/tree/master/highfive/configs, and then nrc should merge the pr and set up the webhook pointing to the right ip address
```

**Comment by manaskarekar** [comment]
Hey @fitzgen does this work right?

**Comment by mgattozzi** [comment]
@manaskarekar you'll need to open up a PR to test that functionality! Even if it's just a test one.

**Comment by manaskarekar** [comment]
@mgattozzi Tried it out, looks like that didn't work. Looking into it.

**Comment by mgattozzi** [comment]
This was fixed and closed during testing with #74. Thanks @manaskarekar! We should be able to now add members for different things/put them on the list now.
